service/service.go

```diff
@@ -326,4 +326,15 @@ func (svc *service) removeExcessEvents() {
 		"amount":   numEventsToDelete,
 		"below_id": deleteEventsBelowId,
 	}).Info("Removed excess events")
+
+	// TODO: REMOVE AFTER 2025-01-01
```

Comment:
2026*, also this reminds me that we can now remove [version tag](https://github.com/getAlby/hub/blob/master/nip47/event_handler.go#L110) too. `GetFirst` also looks deprecated there, I'll create an issue.